src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+
+======= =========================================================================================================
+Example .. code-block:: html 
+
+            <ad:attributes>
+                <ad:attribute>
+                    <ad:attributeName>color</ad:attributeName>
+                    <ad:attributeLocale>nl</ad:attributeLocale>
+                    <ad:attributeLabel>Kleu</ad:attributeLabel>
+                    <ad:attributeValue>Rood</ad:attributeValue>
+                </ad:attribute>
+                <ad:attribute>
+                    <ad:attributeName>color</ad:attributeName>
+                    <ad:attributeLocale>en</ad:attributeLocale>
+                    <ad:attributeLabel>Color</ad:attributeLabel>
+                    <ad:attributeValue>Red</ad:attributeValue>
+                </ad:attribute>
+                <ad:attribute>
+                    <ad:attributeName>Model</ad:attributeName>
+                    <ad:attributeValue>Slim</ad:attributeValue>
+                    <ad:attributeValue>Spro</ad:attributeValue>
+                </ad:attribute>
+            </ad:attributes>
+
+======= =========================================================================================================
+
+.. index:: budgetDetails
+.. _feed_budget:
+
+budgetDetails
+"""""""""""""
+
+Section with budget details
+
+============= ========================================== ========
+Name          Description                                Required
+============= ========================================== ========
+autobid       use auto bidding option true/false         No
+cpc           CPC for the given ad in cents              Yes
+totalBudget   total budget for the given ad in cents     No
+dailyBudget   daily budget for the given ad in cents     No
+============= ========================================== ========
+
+The minimum and maximum values for the total budget depend on the category. 
+If the total budget is not returned with the ad, it means there is an unlimited total budget.
+
+When this value of the daily budget is reached the ad will be paused for the rest of the day. 
+The minimum value depends on the category. Maximum value cannot be higher than the total budget.
+
+The minimum and maximum values of the cost per click (cpc) depend on the category.
+
+======= =========================================================================================================
+Example .. code-block:: html 
+
+            <ad:budget>
+                <ad:totalBudget>5000</ad:totalBudget>
+                <ad:dailyBudget>1000</ad:dailyBudget>
+                <ad:cpc>2</ad:cpc>
+            </ad:budget>
+
+======= =========================================================================================================
+
+.. index:: shippingOptions
+.. _feed_ship:
+
+shippingOptions
+"""""""""""""""
+
+Section with shipping options available for a product.
```

Comment:
I notice that our lingo across is very much `product` suddenly :) but some places there is `ad` instead. Can we keep it consistent, if possible?

Reply:
It sounds like for you "product" and "ad" is the same. I am not there yet :(

Reply:
at this stage, it's hard to distinguish, my personal view is that a product is a type of ad

Reply:
but the language should be consistent